lib/routes.js

```diff
@@ -129,8 +129,25 @@ function addCORS(site) {
  * @returns {express.Router}
  */
 function addSiteController(router, site, providers) {
-  if (site.dir) {
-    const controller = files.tryRequire(site.dir);
+  // fetches itself if it is the main site
+  const parent = siteService.getParentSite(site.slug);
+
+  if (parent && parent.dir) {
+    const controller = files.tryRequire(parent.dir);
+
+    // merge in subsiteController options
+    if (site.subsite) {
+      const subsiteController = files.tryRequire(site.dir);
+
+      if (subsiteController) {
+        // default _.merge overrides values in arrays :facepalm:
+        _.mergeWith(controller, subsiteController, (par, sub) => {
+          if (Array.isArray(par) && Array.isArray(sub)) {
+            return par.concat(sub);
+          }
```

Comment:
needs to `return par` if sub is not an array for whatever reason?

Reply:
Should be ok as is, tested with String params during dev

https://lodash.com/docs/4.17.15#mergeWith

If customizer returns undefined, merging is handled by the method instead. The customizer is invoked with six arguments:
(objValue, srcValue, key, object, source, stack).

Note: This method mutates object.